.github/workflows/flaky.yml

```diff
@@ -11,4 +11,4 @@ jobs:
     - name: Install cargo-flaky
       run: cargo install cargo-flaky
     - name: Run cargo flaky 1000 times
-      run: cargo flaky -i 1 --release
+      run: cd meilisearch-http; cargo flaky -i 1 --release
```

Comment:
```suggestion
      run: |
        cd meilisearch-http
        cargo flaky -i 1 --release
```

Reply:
cargo-flaky has been patched, we now don’t need to cd before running the command

Reply:
We need to create a release for that